packages/components/src/number-control/index.tsx

```diff
@@ -200,6 +221,37 @@ function UnforwardedNumberControl(
 				const baseState = numberControlStateReducer( state, action );
 				return stateReducerProp?.( baseState, action ) ?? baseState;
 			} }
+			size={ size }
+			suffix={
+				size === '__unstable-large' && ! hideHTMLArrows ? (
```

Comment:
The big thing we need to decide before merging is this part!

While I do think the current logic makes sense, I also feel like the big spin buttons should be disabled by default 🤔 They take up _a lot_ of space, so I'm afraid many use cases would prefer to disable them unless they were particularly useful for the specific case. On the other hand, this API could get super messy with the `hideHTMLArrows` and the size variants.

The middle ground I can think of is to update the API so `hideHTMLArrows` is true by default. I _think_ this is a palatable change, given that NumberControl is still experimental and the visual change is non-disruptive. What do you all think?

Reply:
Switching `hideHTMLArrows` from `true` by default works for me 👍 

Another option is to remove `hideHTMLArrows` in favour of something like `spinControls: 'hidden' | 'native' | 'custom'`.

Let me know—happy to defer to you and @ciampo.

Reply:
A `spinControls` prop would've been perfect if we actually had custom spin controls for the other size variants 😂

> Switching `hideHTMLArrows` to `true` by default works for me 👍

Cool. @ciampo ^ We'll go with this if you don't have any objections.

Reply:
Sounds good to me — just to make sure, we will still show these custom, larger spin buttons only for the `__unstable-large` size variant?

Reply:
I changed my mind 😅

I started working on this and in the process appreciated how many instances of `NumberControl` we have in the block editor (quite a few!) and how, in all instances, the arrow buttons / spin buttons are quite handy. I fear if we make `hideHTMLArrows` `true` by default that developers will forget to set `hideHTMLArrows={ false }` and that the UX will suffer. In other words I really think showing arrow buttons / spin buttons is a sensible default.

So I went ahead and implemented a `spinControls` prop as above. The default is `spinControls = 'native'`. I also added some styling so that if `spinControls = 'custom'` and `size = 'small'` things look squished but not totally broken.

I think this makes sense. Let me know if you disagree!

Reply:
Yeah, that makes sense 👍 I really appreciate the effort and care you put into working out the best solution!